test/filename-case.js

```diff
@@ -90,7 +90,8 @@ ruleTester.run('filename-case', rule, {
 		testManyCases('src/foo/fooBar.js', {camelCase: true}),
 		testManyCases('src/foo/FooBar.js', {kebabCase: true, pascalCase: true}),
 		testManyCases('src/foo/___foo_bar.js', {snakeCase: true, pascalCase: true}),
-		testCaseWithOptions('src/foo/bar.js')
+		testCaseWithOptions('src/foo/bar.js'),
+		testCase('src/foo/[bar].js', 'camelCase')
```

Comment:
Can you add some more test-cases? For example:

- `src/foo/{bar}.js`

Reply:
done